cmd/lotus-pcr/main.go

```diff
@@ -448,7 +485,13 @@ func (r *refunder) ProcessTipset(ctx context.Context, tipset *types.TipSet, refu
 			refundValue = types.BigAdd(refundValue, types.BigMul(types.BigDiv(refundValue, types.NewInt(100)), types.NewInt(uint64(r.percentExtra))))
 		}
 
-		log.Debugw("processing message", "method", messageMethod, "cid", msg.Cid, "from", m.From, "to", m.To, "value", m.Value, "gas_fee_cap", m.GasFeeCap, "gas_premium", m.GasPremium, "gas_used", recps[i].GasUsed, "refund", refundValue)
+		gasFees := types.NewInt(0)
+		if r.refundGasFees {
```

Comment:
Ahh, you are refunding all gas fees? I thought we were talking about only refunding gas fees for WindowPoSt